lib/plugins/package/tests/zipService.js

```diff
@@ -120,6 +120,7 @@ describe('#zipService()', () => {
       const unzippedFileData = unzippedData.files;
 
       // binary file is set with chmod of 777
+      console.log(unzippedFileData['bin/some-binary'])
```

Comment:
Faced the same today while reviewing it. Should be resolved with https://github.com/serverless/serverless/pull/1951/commits/8464954e877c6b138b3adbd403dd6da9cc54a604 as otherwise the linter will bark loud.